tests/wpt/README.md

```diff
@@ -121,7 +121,7 @@ use the following:
 
     ./mach create-wpt --reftest tests/wpt/path/to/new/reftest.html --reference tests/wpt/path/to/reference.html
 
-`reference.html` will be created if it doesn't not exist, and `reftest.html`
+`reference.html` will be created if it doesn't exist, and `reftest.html`
```

Comment:
I actually prefer "does not" over "doesn't", since the former is more accepted as the written form.